crates/uv/tests/snapshots/ecosystem__black-lock-file.snap

```diff
@@ -548,7 +548,7 @@ name = "pickleshare"
 version = "0.7.5"
 source = { registry = "https://pypi.org/simple" }
 dependencies = [
-    { name = "pathlib2" },
+    { name = "pathlib2", marker = "python_version == '2.6' or python_version == '2.7' or python_version == '3.2' or python_version == '3.3'" },
```

Comment:
Now that's interesting...

Reply:
(My guess is that it needs to be `python_full_version` to simplify.)

Reply:
Yep! Now it simplifies out entirely.